e2e/internal/kubeclient/kubeclient.go

```diff
@@ -221,3 +221,13 @@ func (c *Kubeclient) logContainerStatus(pod corev1.Pod) {
 		}
 	}
 }
+
+// GetConfigMapData returns the byte representation of the value stored under the provided key in the data section of the ConfigMap.
+func (c *Kubeclient) GetConfigMapData(ctx context.Context, namespace, name, key string) ([]byte, error) {
```

Comment:
I don't think this needs to be a function - it's only two lines effectively, and the error handling is now duplicated here and at the caller. The logging is also not strictly needed, considering that the function returns an error and the test won't poroceed anyway.